### PR TITLE
Feature/exceptions

### PR DESCRIPTION
**Pull request by avion23**
Hello Laroche,

first, thank you for this software. German Taxes are insanity and you have a much better understanding than I.

I'm preparing a pull request with real changes. I'm trying to run my personal transaction reports (https://github.com/avion23/tastyworksTaxes/blob/main/test/merged3.csv) and I have some edge cases like reverse splits and stock mergers.

This Pull Request contains some "harmless" changes like in https://github.com/laroche/tastyworks-pnl/pull/22 . I've extracted this from the later pull requests so you can easier review them.

### CODE REVIEW

File: tw-pnl.py [hunk 1]
```diff
@@ -819,24 +821,24 @@ def check(all_wk, output_summary, output_csv, output_excel, tax_output, show, ve
         if isnan(quantity):
             quantity = 1
         else:
-            if tcode == 'Receive Deliver' and tsubcode in ('Forward Split' or tsubcode == 'Reverse Split'):
```
Comment (avion23) [inline tw-pnl.py:822]: I assume that this was a bug and not intentional

Comment (laroche) [reply tw-pnl.py:822]: Nice spot. Yes, this was a bogus change in b25fd150

**Comment by avion23** [comment]
Okay should be ready now :)

**Comment by avion23** [comment]
Stale — closing.
